do

```diff
@@ -41,6 +41,26 @@ images() {
         --skip=validate "$@"
 }
 
+# This variable is used, but shellcheck can't tell.
+# shellcheck disable=SC2034
+help_dev_images="Build and push the Dev Docker images and manifests."
+dev_images() {
+    set -x
+
+    [[ -f ./bin/goreleaser ]] || install-go-bin "github.com/goreleaser/goreleaser/v2@latest"
+
+    IMAGE_TAG_SUFFIX="-dev-${CIRCLE_BUILD_NUM:-"0"}-$(git rev-parse --short HEAD)"
+
+    SKIP_PUSH="${SKIP_PUSH:-true}" \
+        IMAGE_TAG_SUFFIX="${IMAGE_TAG_SUFFIX:-""}" \
```

Comment:
IMAGE_TAG_SUFFIX will always be defined so we probably do not need the fallback to ""